RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,16 @@
+
+ 1. [ ] (Create release branch)
+ 1. [ ] With each 20XX.1 release, update Copyright notices accordingly
+ 1. [ ] Check wheel install on debian stretch produces an OK error message due to too old python
+ 1. [ ] Write Release Notes
+ 1. [ ] Update Readme
+ 1. [ ] Update Copyright notice if year roll over
+ 1. [ ] Tag commit
+ 1. [ ] Push tag
+ 1. [ ] bump/create demo docker
+ 1. [ ] Wait for CI build to finish (this will upload sdist+wheels to pypi, conda builds to conda cloud)
+ 1. [ ] update morwiki https://morwiki.mpi-magdeburg.mpg.de/morwiki/index.php/PyMOR
+ 1. [ ] update researchgate https://www.researchgate.net/project/pyMOR-Sustainable-Software-for-Model-Order-Reduction
```

Comment:
I thought we would delete the researchgate project for the DFG project?

Reply:
No, we deleted the non-DFG one.

Reply:
@pymor/pymor-devs, I was quite sure we agreed to delete the DFG-one?! What would be the argument for having a DFG-project related thing on researchgate but not for pyMOR itself?

Reply:
I don't remember what was said in the dev meeting, but I would also be for deleting the DFG project and having a general one.

Reply:
We now have the the general one here: https://www.researchgate.net/project/pyMOR-Model-Order-Reduction-with-Python
Sorry for my mix-up.